creator-node/src/services/sync/secondarySyncFromPrimary.ts

```diff
@@ -845,6 +858,30 @@ async function _secondarySyncFromPrimary({
   return { result }
 }
 
+const wasThisNodeEverPrimaryFor = async (
+  wallet: string,
+  libs: any
+): Promise<boolean> => {
+  // Get userId from wallet
+  const users = await libs.User.getUsers(1, 0, null, wallet)
+  const userId = users[0].user_id
+
+  // Return true if any snapshot of the user's history had this node as their primary
+  const history = (
+    await axios({
+      baseURL: libs.discoveryProvider.discoveryProviderEndpoint,
+      url: `/users/history/${userId}`,
+      method: 'get',
+      timeout: 45_000
+    })
+  ).data.data
+  for (const snapshot of history) {
```

Comment:
if a user has 500+ entries we probably don't want to delete this data because we're missing some records. in that case we should return true

Reply:
good call